spec/integration/extensions/active_record_spec.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Dry::Operation::Extensions::ActiveRecord do
+  include Dry::Monads[:result]
+
+  let!(:model) do
```

Comment:
```suggestion
  let(:model) do
```

I'm not very fond of `let!` with bang. Is it going to work without it?